docker/Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM alpine:3.10
 
 ENV BRANCH master
 
+ENV PYTHONPATH "/root/pieman/pieman"
```

Comment:
Maybe it should also be used when running Piemen outside the container?

Reply:
I use it on my machine for testing purposes. In other ways the pieman package should be installed to a virtual environment or globally.
I suggested the changes for the Docker image because there was a lack of consistency. For example, I specify `BRANCH` and what I get is Pieman from that branch, but the pieman package is from PyPI (which is `master`).